deploy/static/provider/aws/deploy-tls-termination.yaml

```diff
@@ -38,11 +38,8 @@ metadata:
   namespace: ingress-nginx
 data:
   allow-snippet-annotations: 'true'
-  http-snippet: |
-    server{
-      listen 2443;
-      return 308 https://$host$request_uri;
-    }
+  http-snippet: "server {\n  listen 2443;\n  return 308 https://$host$request_uri;\n\
```

Comment:
This is still being generated in a broken state :/

Reply:
@rikatz I believe it's fixed in later commits. IIRC I had to modify the yaml in values, but the output now matches the manual output.
https://github.com/kubernetes/ingress-nginx/pull/8000/files#diff-00f078605badc375b2f2f8abbd7e676c1ce529d295f7fd7b214db190c16f8069R326

Reply:
(note that the file you're commenting on doesn't exist in the final commits)    .... i hope :laughing: